openstf/monitoring/teams.py

```diff
@@ -103,6 +107,28 @@ def post_teams_alert(msg, url=None):
     return post_teams(msg, url=url)
 
 
+def build_sql_query_string(df, table):
+    """Build sql insert query string for Teams message output from df.
+
+    Args:
+        df (pd.DataFrame): df of table values to insert in sql
+        table (string): table to insert df into
+
+    Returns:
+        string: sql query string of insert statement
+    """
+    sql_texts = [
+        "INSERT INTO " + table + " (" + str(", ".join(df.columns)) + ") VALUES \n"
+    ]
+    for index, row in df.iterrows():
+        if index != df.index[0]:
+            sql_texts.append(",\n")
```

Comment:
Not needed? 1) Whitelines do not have meaning in sql, and 2) you can remove the `\n` on L121

Reply:
Added it for readability in output message in teams?

Reply:
still not needed if you remove the \n on L121

Reply:
now you get 
`"INSERT INTO table (key, value) VALUES 
('a', 1),
('b', 2)"`
 in your case it would be 
`"INSERT INTO table (key, value) VALUES 
('a', 1),('b', 2)"`
I would prefer option 1 for readability?

Reply:
then the comma fails. You need a comma in between the values. Not at the start and not at the end.